src/lib/apiTypesV2/HttpInfo.cpp

```diff
@@ -91,4 +94,52 @@ std::string HttpInfo::toJson()
   return jh.str();
 }
 
+
+
+/* ****************************************************************************
+*
+* HttpInfo::fill -
+*/
```

Comment:
`using namespace mongo` in commit f810bbf.

Why?
Well, while 'in general' I am still against removing the explicit namespace names, the BSONxxx types have their prefixes and especially, they are used in a million places already in our code, without the explicit namespace.
So ...